src/Test/ListZipperTest.hs

```diff
@@ -320,8 +320,8 @@ moveLeftN'Test =
       moveLeftN' 4 (zipper [3,2,1] 4 [5,6,7]) @?= Left 3
   , testCase "positive in range" $
       moveLeftN' 1 (zipper [3,2,1] 4 [5,6,7]) @?= Right (zipper [2,1] 3 [4,5,6,7])
-  , testProperty "moving zero is `Right . id`" $ \lz ->
-      moveLeftN' 0 lz == (Right . id $ lz :: Either Int (ListZipper Integer))
+  , testProperty "moving zero is `Right`" $ \lz ->
+      moveLeftN' 0 lz == (Right lz :: Either Int (ListZipper Integer))
```

Comment:
I'm uncertain about these; they might be illuminating some law. @tonymorris any opinion?